packages/reports/src/Components/Form/CompositeCalendar/composite-calendar.tsx

```diff
@@ -56,30 +72,30 @@ const CompositeCalendar = props => {
         },
     ]);
 
-    const wrapper_ref = React.useRef();
+    const wrapper_ref = React.useRef() as React.MutableRefObject<HTMLInputElement>;
 
     useOnClickOutside(wrapper_ref, event => {
```

Comment:
This hook accepts 4 args. please make `validationFn` optional in https://github.com/mahdiyeh-deriv/deriv-app/blob/Improvement/72232/Reports_TS_migration/packages/components/src/hooks/use-onclickoutside.ts